tools/ci_build/github/azure-pipelines/linux-migraphx-ci-pipeline.yml

```diff
@@ -38,6 +38,7 @@ jobs:
           --device=/dev/dri \
           --group-add $(video) \
           --group-add $(render) \
+          --user $UID:$(id -g $USER) \
```

Comment:
It is needed because in migraphx-ci-pipeline-env.Dockerfile it doesn't set uid. The other docker files have:

```
ARG BUILD_UID=1001
ARG BUILD_USER=onnxruntimedev
RUN adduser --uid $BUILD_UID $BUILD_USER
WORKDIR /home/$BUILD_USER
USER $BUILD_USER
```